packages/outstatic/src/utils/createCommitInput.ts

```diff
@@ -47,14 +49,23 @@ export const createCommitInput = ({
             .replace(/[^a-zA-Z0-9-_\.]/g, '-')
             .replace(/(\.[^\.]*)?$/, `-${randString}$1`)
 
+          const filePath = (() => {
+            switch (type) {
+              case 'images':
+                return IMAGES_PATH
+              default:
+                assertUnreachable(type)
+            }
+          })()
```

Comment:
I could turn this into more concise, yet less future-proof:

```suggestion
          const filePath =
            type === 'images' ? IMAGES_PATH : assertUnreachable(type)
```

I think it would be nice to have a `match` function that would act the same as `switch`, but work as an expression, so this would be possible:

```ts
const filePath = match(type, {
  images: IMAGES_PATH
})
```

Reply:
Creating a match function would be cool. I'd go for a more explicit name like `matchTypeToPath`. It's up to you. The current switch statement is clear enough, and as you've mentioned, future proof.

Reply:
I'm good